src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+function normalizeNumber(str, separator) {
+  let normalizeStr = '';
+  const reversedStr = str.split('').reverse().join('');
+
+  for (let i = 0; i < reversedStr.length; i++) {
+    if (i % 3 === 0 && i !== 0) {
+      normalizeStr += separator;
+    }
+    normalizeStr += reversedStr[i];
+  }
+
+  return normalizeStr.split('').reverse().join('');
+}
+
+const countriesPopulation = document.querySelectorAll('.population');
+const total = [...countriesPopulation]
+  .map(item => +item.textContent.split(',').join(''))
+  .reduce((a, b) => a + b)
+  .toString();
```

Comment:
Do you need here convert to string?

```suggestion
const total = [...countriesPopulation]
  .map(item => +item.textContent.split(',').join(''))
  .reduce((a, b) => a + b);
```